### PR TITLE
[minor] Support MVI integration with Object Storage in oneclick_add_visualinspection

### PR DESCRIPTION
**Pull request by andrercm**
Fix for:

- https://github.com/ibm-mas/ansible-devops/issues/1062

### CODE REVIEW

File: ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml [hunk 1]
```diff
@@ -25,8 +33,30 @@
...
+        - configure_cos
+        - cos_type in ['ibm','ocs']
+    - include_role:
+      # Configure COS bucket in IBM Cloud Object Storage (10 min)
+        name: ibm.mas_devops.cos_bucket
+      when:
+        - mas_app_settings_visualinspection_object_storage_enabled
+        - mas_app_settings_visualinspection_object_storage_workspace
+        - configure_cos_bucket
+        - cos_type in ['ibm','aws']
```
Comment (dclain) [inline ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml:50]: just wondering why we're checking different cos types , `['ibm', 'aws']` here vs `['ibm,'cos']` in the previous role?

Comment (andrercm) [reply ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml:50]: With this we are allowing options for users to also provision object storage if they don't have one setup yet.
While using `cos` role, user will be able to configure either `ocs` or `ibm cloud object storage` instances to be used.
Whereas while using the `cos_bucket` role, users will be able to create a bucket within the ibm cloud object storage instance or in aws/s3 (in aws, users can allocate a bucket directly without the need of a COS instance per say). 

If `cos_type` is different from these two options, then either user is using `ocs` which don't required a bucket to be created, or won't need a bucket at all to be created using the automation i.e user might have one bucket already configured.

File: ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml [hunk 2]
```diff
@@ -25,8 +33,36 @@
           - lookup('env', 'MAS_CONFIG_DIR') != ""
           - lookup('env', 'IBM_ENTITLEMENT_KEY') != ""
         fail_msg: "One or more required environment variables are not defined"
-
+    - include_role:
+      # Configure Object Storage (15 min)
+      # With this we are allowing options for users to also provision Cbject Storage instance if they don't have one setup yet
```
Comment (dclain) [inline ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml:38]: typo

File: ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml [hunk 3]
```diff
@@ -25,8 +33,36 @@
...
+      # While using cos role, users will be able to configure either OCS/ODF or IBM Cloud Object Storage instances to be used
+      # This will also generate the ObjectStorateCfg file that will be used to configure the target Object Storage instance
```
Comment (dclain) [inline ibm/mas_devops/playbooks/oneclick_add_visualinspection.yml:40]: typo

**Review by dclain — APPROVED** [review]

**Review by durera — APPROVED** [review]

**Comment by durera** [comment]
Want to get a new patch release out first before merging this new feature into master.

**Pull request merged**